components/meter/meter-circle.js

```diff
@@ -57,26 +74,33 @@ class MeterCircle extends MeterMixin(RtlMixin(LitElement)) {
 
 		const primary = this._primary(this.value, this.max) || '';
 		const secondary = this._secondary(this.value, this.max, this.text);
+		const secondaryTextElement = (this.text && !this.textHidden) ? html`<div class="d2l-body-small d2l-meter-circle-text-secondary">${secondary}</div>` : nothing;
```

Comment:
there are 3 uses where we need to add text-hidden before we merge this https://d2l.slack.com/archives/C0PHG3QB0/p1712758960144389?thread_ts=1712758381.367159&cid=C0PHG3QB0